### DISCUSSION TITLE
Addressing multiple TotalMix instances in the network - TotalMix Plugin Classic

### DISCUSSION THREAD

**Discussion by shells-dw** [shells-dw/streamdeck-totalmix#22] [Announcements]
Hi,
I just wanted to make this a bit more visible, hopefully people stumble over this Discussions thing here.
If you're TL;DR:  [I've published a "Classic" version](https://github.com/shells-dw/streamdeck-totalmix-classic/releases) of the last v2.x release on GitHub.

I've removed the ability to address different TotalMix instances in the network with version 3.x and the overhaul/rewrite of the mirroring-function as some of you noticed.

Due to how I handle the OSC communication with TotalMix now, it would have been quite complex (and most certainly bad for the performance) to implement the level of flexibility the older versions offered. The upside is, that the vast majority of users benefit from a way more robust and quick mirroring.

The old version had quite a few bugs and issues with Windows 11 I wanted to fix. Due to that I had to change the OSC library the plugin uses - as things go, once you touch something, everything breaks and you end up with a more-or-less total rewrite.
I migrated the plugin over to the Loupedeck platform - and had a couple of fresh ideas from that development I wanted to bring over to the StreamDeck version and finally bring in the fixes and improvements that were needed.

I'm sure you're also aware, that I'm coding this thing here in my free time just for fun and taking ownership, since I moved on from RME to UAD devices (for everything else than audio on my gaming-pc) years ago.

Long story short - I'm not really into bringing the (so I claim, your POV may vary) niche-application of addressing multiple TotalMix instances back and put another dozen hours into another plugin overhaul for a feature just a handful of users use. _unless you want to contract me for doing that 😉_

If you rely on addressing multiple TotalMix instances in the network, [I've published a "Classic" version](https://github.com/shells-dw/streamdeck-totalmix-classic/releases) of the last v2.x release here on GitHub that was on the StreamDeck store since Summer last year. I've changed the plugin name and identifier so it will install alongside _this_ version of the plugin and not be overwritten by any upcoming updates.

Thanks for your understanding.
